lib/client.js

```diff
@@ -22,7 +22,11 @@ var clientOptions = require('./client-options');
  *  Only one contact point is required (the driver will retrieve the address of the other nodes automatically),
  *  but it is usually a good idea to provide more than one contact point, because if that single contact point is unavailable, the driver will not be able to initialize correctly.
  * </p>
- * @property {String} keyspace
+ * @property {String} keyspace The logged keyspace for all the connections created within the {@link Client} instance.
+ * @property {Number} refreshSchemaDelay The default window size in milliseconds used to debounce node list and schema
+ * refresh metadata requests. Default: 1000.
+ * @property {Boolean} isMetadataEnabled Determines whether client-side schema metadata retrieval and update is enabled.
```

Comment:
I took the name from the java driver but Im not very fond of it neither, `isMetadataSyncEnabled` sounds better, I'll change it.